composer.json

```diff
@@ -63,7 +63,6 @@
         "drupal/twig_tweak": "^3.1",
         "drupal/views_ajax_history": "^1.6",
         "drush/drush": "^10.0",
```

Comment:
No longer need to install `emulsify-drupal` via composer. When the setup process runs we tell EmulsifyCLI to setup our theme for us